test/Benchmarks/Exporter/JaegerExporterBenchmarks.cs

```diff
@@ -155,13 +119,25 @@ private Activity CreateTestActivity()
                     ActivityTraceFlags.Recorded)),
             };
 
-            return activitySource.StartActivity(
```

Comment:
This was returning null before because there was nothing sampling the source, which caused the benchmarks to blow up. ActivitySource API scoffs at the pit of success! 🕳️